simulator/src/main.rs

```diff
@@ -87,7 +83,7 @@ enum Mode {
     Continuous(Continuous),
 
     /// Run in json mode, behaviour is defined by the file given by --path
-    Defined(Defined),
+    SimulationConfig(SimulationConfig),
```

Comment:
This was fine as it was, as this is for the CLI. The change was specific to the name of the `simulation_config` module.